cumulusci/tasks/metadata_etl/layouts.py

```diff
@@ -68,3 +68,70 @@ def _create_new_related_list(self, metadata, api_name, related_list):
             elem.append("customButtons", button)
 
         elem.append("relatedList", text=related_list)
+
+
+class AddFields(MetadataSingleEntityTransformTask):
+    entity = "Layout"
+    task_options = {
+        "layout_section": {
+            "description": "Which layout section the fields should be added to, default is Information",
+            "required": False,
+        },
+        "fields": {
+            "description": "Array of field API names to append to specified layout section",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _transform_entity(
+        self, metadata: MetadataElement, api_name: str
+    ) -> Optional[MetadataElement]:
+
+        # Set layout section and default to "Information" section if not passed in
+        layout_section = self._inject_namespace(self.options.get("layout_section") or "Information")
+
+        # Set fields to be added to layout section
+        adding_fields = [
+            self._inject_namespace(field)
+            for field in process_list_arg(self.options.get("fields", []))
+        ]
+
+        self._remove_existing_fields(metadata, api_name, adding_fields)
+
+        self._set_adding_fields(metadata, api_name, layout_section, adding_fields)
+
+        return metadata
+
+    ## If fields passed in that already exist in the Page Layout, remove from current sections
```

Comment:
I'd be interested to hear more about this piece. In many Metadata ETL situations, we deliberately do nothing if the configuration already exists. For example, if the field is already on the page layout, moving it to the end of a section _seems_ relatively unlikely to me to be the right outcome. Is there a specific use case that needs this mechanic, as opposed to simply leaving it where it is?